src/ebmc/property_checker.h

```diff
@@ -12,6 +12,13 @@ Author: Daniel Kroening, dkr@amazon.com
 #include "ebmc_properties.h"
 #include "transition_system.h"
 
+enum class property_checker_resultt
+{
+  REPORT_RESULT,
```

Comment:
I am struggling to understand what this one is about? Or, rather, why is "SUCCESS" not a "result?"

Reply:
It's not a verification result -- rename REPORT_RESULT -> REPORT_VERIFICATION_RESULT?

Reply:
Or simply `VERIFICATION_RESULT`?

Reply:
`VERIFICATION_RESULT` works for me.

Reply:
done